.github/workflows/pull-request-test.yml

```diff
@@ -94,6 +94,7 @@ jobs:
       run: |
         echo "$RENKUBOT_KUBECONFIG" > renkubot-kube.config
         helm repo add stable 	https://kubernetes-charts.storage.googleapis.com
+        helm repo add codecentric https://codecentric.github.io/helm-charts
```

Comment:
This should be added to instructions that describe deploying renku from source - but I'm not sure it belongs in this PR?

Reply:
Yes, documentation should be updated.
I also did not included it here because I was not sure it belongs in this PR but I can certainly add it. Bitnami repo should also be added to the docs.

Reply:
sure that's fine - please make an issue for it. Thanks!